source/faucet/main.d

```diff
@@ -533,7 +533,9 @@ public class Faucet : FaucetAPI
             .draw(amount, [pubkey])
             .sign(freeze ? OutputType.Freeze : OutputType.Payment);
         log.info("Sending {} BOA to {}", amount, recv);
-        this.randomClient().postTransaction(tx);
+        TransactionResult result = this.randomClient().postTransaction(tx);
+        ensure(result.status == TransactionResult.Status.Accepted,
```

Comment:
Might need to put after following lines (i.e. `increaseMetricBy`); couldn't decide.